Microsoft.DotNet.Interactive.CSharp/CSharpKernelExtensions.cs

```diff
@@ -216,8 +217,8 @@ public static string GetDisplayValueId(PackageReference package)
                             context.Publish(new PackageAdded(resolvedReference));
 
                             // Load extensions
-                            await context.HandlingKernel.SendAsync(
-                                new LoadExtensionsInDirectory(
+                            await context.HandlingKernel.GetRoot().SendAsync(
+                                new LoadKernelExtensionsInDirectory(
                                     resolvedReference.PackageRoot,
                                     addedAssemblyPaths));
```

Comment:
We're doing away with the assembly paths, yeah?

Reply:
we will only require the packageRoot once we finish